### ISSUE TITLE
vmware exporter reports as not successful but metrics are reported (v0.30.0-rc.1)

### ISSUE THREAD

**Issue by aa-matthias** [prometheus-community/windows_exporter#1788]
### Current Behavior

After testing v0.30.0-rc.1 (#1774 ✌️) vmware collector is reporting no success but metrics are collected.

```
windows_exporter_collector_success{collector="vmware"} 0
windows_exporter_collector_timeout{collector="vmware"} 0
windows_exporter_collector_duration_seconds{collector="vmware"} 0
```

but:

```
windows_vmware_host_processor_speed_mhz 2594
```

### Expected Behavior

shouldn't it have been successful? I cannot see any errors in the windows event log as well.

### Environment

- windows_exporter Version: 0.30.0-rc.1
- Windows Server Version: 2019

### windows_exporter logs

```shell
nothing beeing logged in eventlog
```

**Comment by aa-matthias** [comment]
I do see that memory metrics are missing, only cpu collector is reporting anything.

**Comment by jkroepke** [comment]
While refactor the vmware collector, there was a lot of guessing, since I found no docs.

The error should be logged on DEBUG level and on application start. 

Could you run typeperf -qx to generate a list of metrics and attach it?

**Comment by aa-matthias** [comment]
not sure but i think these are the ones (german locale here, sorry).

```txt
\VM-Prozessor(_Total)\Limit in MHz
\VM-Prozessor(_Total)\Reservierung in MHz
\VM-Prozessor(_Total)\Anteile
\VM-Prozessor(_Total)\CPU stolen time
\VM-Prozessor(_Total)\Prozessorzeit (%)
\VM-Prozessor(_Total)\Effektive VM-Geschwindigkeit in MHz
\VM-Prozessor(_Total)\Hostprozessorgeschwindigkeit in MHz
\VM-Arbeitsspeicher\Aktiver Arbeitsspeicher in MB
\VM-Arbeitsspeicher\Arbeitsspeicher-Balloon in MB
\VM-Arbeitsspeicher\Arbeitsspeicherlimit in MB
\VM-Arbeitsspeicher\Zugeordneter Arbeitsspeicher in MB
\VM-Arbeitsspeicher\Arbeitsspeicher-Overhead in MB
\VM-Arbeitsspeicher\Arbeitsspeicherreservierung in MB
\VM-Arbeitsspeicher\Gemeinsamer Arbeitsspeicher in MB
\VM-Arbeitsspeicher\Eingesparter gemeinsamer Arbeitsspeicher in MB
\VM-Arbeitsspeicher\Arbeitsspeicheranteile
\VM-Arbeitsspeicher\Abgerufener Auslagerungsspeicher in MB
\VM-Arbeitsspeicher\Memory Target Size
\VM-Arbeitsspeicher\Belegter Arbeitsspeicher in MB
```

**Comment by aa-matthias** [comment]
confirmed to be fixed with v0.30.0-rc.2
